guides/hack/13-contexts-and-capabilities/02-local-operations.md

```diff
@@ -10,7 +10,7 @@ Consider the following potential (although not necessarily planned) contexts (wi
 * `write_prop`, representing the permission to mutate properties of objects
 * `dynamic`, representing the permission to cast a value to the `dynamic` type
 
-In all of the below cases, the relevant local operations are only legal due to the existence of the matching capabilities within the context of the function. In the world where all these contexts (and matching capabilities exist), some or all may be included within the `defaults` context.
+In all of the below cases, the relevant local operations are only legal due to the existence of the matching capabilities within the context of the function.
```

Comment:
This is definitely an improvement too, but perhaps we should avoid 'legal'? How about this?

```suggestion
In all of the below cases, the relevant local operations typecheck due to the matching capabilities within the context of the function.
```